docs/source/apis/aws_marketplace.rst

```diff
@@ -0,0 +1,245 @@
+
+   - Go to “Route tables” in the VPC dashboard.
+   - Enter a ``Name``.
+   - Select the ``VPC`` from Step 2 above.
+   - Click ``Create route table``
+
+.. image:: imgs/AWS/VPC_Step5.png
+  :align: center
+  :alt: create route table
+
+6. **Connecting Route Tables and Internet Gateway**:
+
+   - Go to the VPC set up in Step 2.
+   - Connect the **public subnets** to the **route table** from Step 5.
+
+     - Select the public subnet from Step 3.
+     - Click ``Actions`` button on the top right-hand corner
+     - Select ``Edit route table association``
+     - Repeat the process for the two public subnets.
+
+   - Connect the Route table to Internet Gateway
+
+     - Click the ``route table`` from VPC Details page.
+     - Click ``Edit route``
+     - Click ``Add route``:
+
+   - For the **private subnet**, use the main route table or create a new one without a route to the internet gateway.
+
+.. image:: imgs/AWS/VPC_Step6.png
+  :align: center
+  :alt: connect public subnet to route table
+
+.. image:: imgs/AWS/VPC_Step7.png
+  :align: center
+  :alt: VPC Resource Maps
+
+7. **Inspect VPC Resource Map**:
+
+   You can check the configurations from the Resource Maps on the VPC Details dashboard.
+
+Part 2: Deploying Unstructured API from AWS Marketplace
+-------------------------------------------------------
+
+8. **Visit the Unstructured API page on AWS Marketplace**
+
+   - Link: `Unstructured API Marketplace <http://aws.amazon.com/marketplace/pp/prodview-fuvslrofyuato>`_.
+   - Click ``Continue to subscribe``
+   - Review Terms and Conditions
+   - Click ``Continue to Configuration``
+
+.. image:: imgs/AWS/Marketplace_Step8.png
+  :align: center
+  :alt: Unstructured API on AWS Marketplace
+
+9. **Configure the CloudFormation**
+
+   - Select ``CloudFormation Template`` from the Fulfillment option dropdown menu.
+   - Use the default ``Unstructured API`` template and software version.
+   - Select the ``Region``
+
+     - *Note: select the same region where you set up the VPC in Part 1.*
+   - Click ``Continue to Launch`` button.
+   - Select ``Launch CloudFormation`` from Choose Action dropdown menu.
+   - Click ``Launch`` button.
+
+
+.. image:: imgs/AWS/Marketplace_Step9.png
+  :align: center
+  :alt: CloudFormation Configuration
+
+
+10. **Create Stack on CloudFormation**
+
+    The Launch button will redirect to ``Create stack`` workflow in the CloudFormation.
+
+    **Step 1: Create stack**
+
+    - Select the ``Template is ready``
+    - Use the default template source from ``Amazon S3 URL``
+    - Click ``Next`` button.
+
+    .. image:: imgs/AWS/Marketplace_Step10a.png
+        :align: center
+        :alt: Create Stack
+
+
+    **Step 2: Specify stack details**
+
+    - Provide ``stack name``
+    - In the **Parameters** section, provide the ``KeyName``, ``Subnets``, and ``VPC`` from Part 1 above.
+    - Specify, ``LoadBalancerScheme`` to **internet-facing** and ``SSHLocation`` to  **0.0.0.0/0**
+    - Click ``Next`` button.
```

Comment:
can you please make each of these steps in the order they appear on the page? one step for each item